.travis.yml

```diff
@@ -3,12 +3,15 @@ dist: xenial
 
 language: python
 
+services:
+  - mongodb
```

Comment:
is this useful or 
+ should we add pymongodb to tox 
or 
should we ask people to install mongodb on their laptop ?

This is important for the installation instructions on the readme

Reply:
I think you could just get rid of it!